### DISCUSSION TITLE
第5节课作业(4班)

### DISCUSSION THREAD

**Discussion by FlamingoPg** [InternLM/Tutorial#316] [作业]
请大家把第五节课作业交到这里哦~

**Comment by zhjunqin** [comment]
zhjunqin: https://github.com/zhjunqin/InternLM-tutorial/tree/master/Fifth

**Comment by onlyzer0** [comment]
onlyzer0:https://blog.csdn.net/qq_42907812/article/details/135611313?spm=1001.2014.3001.5501

**Comment by phbst** [comment]
https://thankful-archeology-738.notion.site/79aaa431285a46a4b79782e18650eb0f?pvs=4
